packages/fortinet_fortimail/data_stream/log/elasticsearch/ingest_pipeline/pipeline_encryption.yml

```
@@ -0,0 +1,56 @@
+---
+description: Pipeline for processing Encryption logs.
+processors:
+  - rename:
+      field: fortinet_fortimail.log.log_details.session_id
+      target_field: fortinet_fortimail.log.session_id
+      ignore_missing: true
+  - grok:
+      field: message
+      patterns:
+        - '^%{DATA}[U|u]ser %{NOTSPACE:_tmp.user} %{DATA}%{IP:fortinet_fortimail.log.ip}%{GREEDYDATA:_tmp.msg}$'
+        - "^%{DATA}[U|u]ser %{NOTSPACE:_tmp.user} %{GREEDYDATA:_tmp.msg},%{SPACE}sent from:%{SPACE}\\'%{DATA:fortinet_fortimail.log.sent_from}\\',%{SPACE}subject:%{SPACE}\\'%{GREEDYDATA:_tmp.subject}\\'%{SPACE}%{GREEDYDATA:_tmp.msg2}$"
+        - "^%{DATA}[U|u]ser %{NOTSPACE:_tmp.user} %{GREEDYDATA:_tmp.msg},%{SPACE}sent from:%{SPACE}\\'%{DATA:fortinet_fortimail.log.sent_from}\\',%{SPACE}subject:%{SPACE}\\'%{GREEDYDATA:_tmp.subject}\\'$"
+        - '^%{DATA}[U|u]ser %{NOTSPACE:_tmp.user} %{GREEDYDATA:_tmp.msg}$'
```

```suggestion
        - '^%{DATA}[Uu]ser %{NOTSPACE:_tmp.user} %{DATA}%{IP:fortinet_fortimail.log.ip}%{GREEDYDATA:_tmp.msg}$'
        - "^%{DATA}[Uu]ser %{NOTSPACE:_tmp.user} %{GREEDYDATA:_tmp.msg},%{SPACE}sent from:%{SPACE}\\'%{DATA:fortinet_fortimail.log.sent_from}\\',%{SPACE}subject:%{SPACE}\\'%{GREEDYDATA:_tmp.subject}\\'(?:%{SPACE}%{GREEDYDATA:_tmp.msg2})?$"
        - '^%{DATA}[Uu]ser %{NOTSPACE:_tmp.user} %{GREEDYDATA:_tmp.msg}$'
```
(untested)
Could also consider using `(?i)` in place of `[Uu]` (not `[U|u]` which matches `U`, `u` and `|`), but this depends on the remainder of the expectations.